message.go

```diff
@@ -152,8 +156,48 @@ func (this *CommandMsg) pushiOS(server *Server) {
 	alert, _ := pn.PayloadString()
 
 	if resp.Error != nil {
-		log.Printf("Alert: %s\n", alert)
-		log.Printf("Error: %s\n", resp.Error)
+		log.Printf("Alert (iOS): %s\n", alert)
+		log.Printf("Error (iOS): %s\n", resp.Error)
+	}
+}
+
+func (this *CommandMsg) pushAndroid(server *Server) {
+	registration_ids, registration_ids_ok := this.Command["registration_ids"]
+
+	if !registration_ids_ok {
+		log.Println("Registration ID(s) not provided!")
+		return
+	}
+
+	msg, err := this.formatMessage()
+	if err != nil {
+		log.Println("Could not format message")
+		return
+	}
+
+	data := map[string]interface{}{"event": msg.Event, "data": msg.Data, "time": msg.Time}
+
+	regIDs := strings.Split(registration_ids, ",")
+	gcmMessage := gcm.NewMessage(data, regIDs...)
```

Comment:
what does the `...` do as a parameter? I've only seen it in function declarations.